package.json

```diff
@@ -22,10 +22,5 @@
       "@platforma-sdk/workflow-tengo": "file:/Users/poslavskysv/Projects/milab/platforma/platforma-sdk/sdk/workflow-tengo/package.tgz",
       "@milaboratories/uikit": "file:/Users/poslavskysv/Projects/milab/platforma/platforma-sdk/lib/ui/uikit/package.tgz"
     }
-  },
-  "overrides": {
```

Comment:
vite update was released, removing this fix until everyone forgot it was here and caused some trouble